### DISCUSSION TITLE
Problem compiling the example DMXWrite.

### DISCUSSION THREAD

**Discussion by rodcoles** [someweisguy/esp_dmx#116] [General]
*** NOT AN ISSUE *** I had ESP32C3 chosen instead of ESP32S3 - now ver 3 compiles fine. Unfortunately, the easiest way to see my mistakes, is to post that it is someone else's issue - duh! Thanks again for all you are doing! 

Hi Mitch,

Thanks for all the work, love what you have done, and the great documentation! I am trying move my stuff to the ESP32-S3 board, and want to test your DMXWrite example, I am getting the following errors - any thoughts on what I can do?

c:\Users\rodco\Documents\Arduino\libraries\esp_dmx\src\dmx\hal\timer.c: In function 'dmx_timer_init':
c:\Users\rodco\Documents\Arduino\libraries\esp_dmx\src\dmx\hal\timer.c:97:52: error: 'isr_handle' undeclared (first use in this function); did you mean 'intr_handle_t'?
   timer_isr_callback_add(timer->group, timer->idx, isr_handle, isr_context,
                                                    ^~~~~~~~~~
                                                    intr_handle_t
c:\Users\rodco\Documents\Arduino\libraries\esp_dmx\src\dmx\hal\timer.c:97:52: note: each undeclared identifier is reported only once for each function it appears in

Using library esp_dmx at version 3.1.0 in folder: C:\Users\rodco\Documents\Arduino\libraries\esp_dmx 
exit status 1

Compilation error: exit status 1

Thanks,
Rod

**Comment by someweisguy** [comment]
Glad to hear you got this figured out! :)

Let me know if you have any other questions!
